src/main/java/seedu/todo/commons/util/DateUtil.java

```diff
@@ -13,6 +13,14 @@
  */
 public class DateUtil {
 
+    private static final String FROM_NOW = "later";
+    private static final String TILL_NOW = "ago";
+    private static final String SPACE  = " ";
```

Comment:
Don't really like having to declare a space as a magic string, maybe use `String.format` in those cases.